python/lsst/ts/scheduler/driver/feature_scheduler.py

```diff
@@ -74,6 +81,10 @@ class FeatureSchedulerParameters(DriverParameters):
 class FeatureScheduler(Driver):
```

Comment:
Please add a type definition for `Driver` and a return type for the method.

A gentle suggestion to add this line to the `mypy` section in `setup.cfg` so MyPy complains about these missing definitions. It will force you to add type definitions to the test files as well though.

disallow_untyped_defs = True